pkg/dsl/process/handle.go

```diff
@@ -146,7 +146,7 @@ func (ctx *templateContext) BaseContextFile() string {
 
 	if len(ctx.configs) > 0 {
 		bt, _ := json.Marshal(ctx.configs)
-		buff += ConfigFieldName + ": " + string(bt)
+		buff += ConfigFieldName + ": " + string(bt) + "\n"
```

Comment:
good catch, please also add a new line for outputSecretName in  L160

Reply:
ok